### PR TITLE
Add troubleshooting info when building a project

### PR DESCRIPTION
**Pull request by vamelchenia**
## Description

I tried building the project, install all dependencies and encountered an issue with ruby versions that didn't allow me to proceed with building. We fixed the issue with @dricazenck and decided that I update README file with this info.

## Change Type
- [x] Documentation

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] I checked and followed the [contributor guide](CONTRIBUTING.md) 
- [x] I have tested my changes locally.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,6 +55,32 @@ We recommend using Bundler to install and run Jekyll. Bundler manages Ruby gem d
 * [Test locally with Jekyll](https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/testing-your-github-pages-site-locally-with-jekyll)
 
 
+## How to Troubleshoot Building Project Locally
+
+The project requires to use ruby>=3.1.*. Make sure that your system looks at the correct version of ruby.
+
```
Comment (dricazenck) [inline README.md:61]:
```suggestion
### Setup ruby with rbenv 
```

File: README.md [hunk 2]
```diff
@@ -55,6 +55,32 @@ We recommend using Bundler to install and run Jekyll. Bundler manages Ruby gem d
 * [Test locally with Jekyll](https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/testing-your-github-pages-site-locally-with-jekyll)
 
 
+## How to Troubleshoot Building Project Locally
```
Comment (dricazenck) [inline README.md:58]: Could you add here some of the errors that you are trying to fix? 

So, the error is when `bundle install` and the exception it is (could be the screenshots) you shared with me.  

Also another step to try is to remove the `Gemfile.lock`

Comment (vamelchenia) [reply README.md:58]: Done

File: README.md (hunk 1 above)
Comment (vamelchenia) [reply README.md:61]: Done

**Review by dricazenck — APPROVED** [review]

**Pull request merged**